server/graphql/lib.rs

```diff
@@ -11,6 +11,7 @@ use async_graphql::http::{playground_source, GraphQLPlaygroundConfig};
 use async_graphql::{EmptyMutation, EmptySubscription, Schema};
 use async_graphql::{MergedObject, Response};
 use async_graphql_actix_web::{GraphQLRequest, GraphQLResponse};
+use graphql_asset_catalogue::AssetCatalogueItemQueries;
```

Comment:
```suggestion
use graphql_asset_catalogue::AssetCatalogueQueries;
```
I thinik `Item` is too specific?